x/exchange/spec/03_messages.md

```
@@ -299,6 +300,26 @@ It is expected to fail if:
 +++ https://github.com/provenance-io/provenance/blob/v1.20.0/proto/provenance/exchange/v1/tx.proto#L317-L318
 
 
+### MarketTransferCommitments
+
+A market can transfer funds from one market to another using the `MarketTransferCommitments` endpoint.
+The `admin` must have the `PERMISSION_CANCEL` permission in the market (or be the `authority`).
+
+It is expected to fail if:
+* The currentmarket does not exist.
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: dash; Actual: asterisk

(MD004, ul-style)